cpp/oneapi/dal/backend/primitives/reduction/reduction_rm_cw_dpc.hpp

```diff
@@ -58,46 +58,6 @@ class reduction_rm_cw_naive {
     const std::int64_t wg_;
 };
 
-template <typename Float, typename BinaryOp, typename UnaryOp>
-class kernel_reduction_rm_cw_naive_local;
-
-template <typename Float, typename BinaryOp, typename UnaryOp>
```

Comment:
I am not sure we need to remove this type of reduction. I understand that this type of reduction is not necessary for now, we can live without it. But cannot you imagine some scenarios when it is useful? And it is implemented already. I am just not sure we need to throw away this code.

Reply:
FIxed